app/src/main/java/com/example/drawingApp/CustomViews/ColorSlider.kt

```diff
@@ -0,0 +1,119 @@
+package com.example.drawingApp.CustomViews
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.children
+import kotlin.math.max
+import kotlin.math.min
+
+@SuppressLint("ClickableViewAccessibility")
+class ColorSlider @JvmOverloads constructor(
```

Comment:
This doesn't look appropriate to this ticket

Reply:
I think this was due to previous conflicts